charts/pulsar/templates/bookkeeper-statefulset.yaml

```diff
@@ -162,6 +162,9 @@ spec:
         command: ["sh", "-c"]
         args:
         - >
+        {{- if .Values.bookkeeper.additionalCommand }}
+          {{ .Values.bookkeeper.additionalCommand }}
+        {{- end }}
```

Comment:
Did you verify that this renders properly using `helm template`? I copied this into my repo and seem to be getting an error.

Reply:
Yes it renders properly for me @michaeljmarshall. I'm running Helm v3.6.3

```shell
# helm version --short
v3.6.3+gd506314

# helm template pulsar/ | tail
# software distributed under the License is distributed on an
# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
# KIND, either express or implied.  See the License for the
# specific language governing permissions and limitations
# under the License.
#

# deploy zookeeper only when `components.zookeeper` is true

# define the storage class for data directory

# echo $?
0
```